server/engine/rooms.js

```diff
@@ -1,3 +1,5 @@
+const users = require("./users");
+
```

Comment:
Don't know the operator assignment shortcut for this offhand and it might make the code harder to read. Should we just turn off this rule?